routes/comments.js

```diff
@@ -0,0 +1,53 @@
+var express = require("express");
```

Comment:
Should this be `let`?
```suggestion 
let express = require("express");
```